### ISSUE TITLE
Failure to parse param-names

### ISSUE THREAD

**Issue by jengelh** [libical/libical#410]
libical up to and including 3.0.6 illegally rejects (when calling `Parse_MIME`) most parameter values:
```
BEGIN:VCARD
EMAIL;PREF=1:otto@test.net
END:VCARD
```
```
BEGIN:VCARD
EMAIL;PREF="1":otto@test.net
END:VCARD
```
It only appears to accept param-values when there is a leading alpha char.
```
BEGIN:VCARD
EMAIL;PREF=A1:otto@test.net
END:VCARD
```
RFC 6350 on the matter:
```
any-param  = (iana-token / x-name) "=" param-value *("," param-value)
param-value = *SAFE-CHAR / DQUOTE *QSAFE-CHAR DQUOTE
SAFE-CHAR = WSP / "!" / %x23-39 / %x3C-7E / NON-ASCII
```

**Comment by winterz** [comment]
our new libicalvcard library coming in the 4.0 release (currently the master branch) supports RFC6350.

"The PREF parameter is OPTIONAL and is used to indicate that the
   corresponding instance of a property is preferred by the vCard
   author.  Its value MUST be an integer between 1 and 100 that
   quantifies the level of preference.  Lower values correspond to a
   higher level of preference, with 1 being most preferred."


once 4.0 lands, we'll be deprecating anything to do with vcard support in the libicalvcal library
so users should start thinking about porting.

@ksmurchison anything to add here?

**Comment by ksmurchison** [comment]
Not much to add. I _think_ libicalvcard is in better in that it's more complete and uses a similar API to iCalendar support

**Comment by winterz** [comment]
will close this then.
=>
won't fix for 3.x
and use libicalvcard in version 4

**Comment by jengelh** [comment]
We're no longer using libical.
